substreams/substreams.yaml

```diff
@@ -1,6 +1,6 @@
 specVersion: v0.1.0
 package:
-  name: aptos
```

Comment:
Why the package name change? For me it's substream's name and should be the chain name directly, `ethereum`, `near`, `solana`. We never added suffix `_firehose`

Reply:
Ah got ya. I was hoping that the generated .spkg file would say aptos-firehose since we also have aptos-substreams. But I'll respect the pattern.